members/B000944.yaml

```diff
@@ -4,63 +4,65 @@ contact_form:
   steps:
     - visit: "http://www.brown.senate.gov/contact/email"
     - find:
-      - selector: form#contactForm
+      - selector: "#form-466BAE0A-C4F2-8C2C-A3E7-92224410B0FB"
```

Comment:
these generated selectors seem quite brittle, are there class selectors or something that could be used instead?

Reply:
I couldn't find anything more reliable that those selectors, no 😢